docs/sql/tutorials/house-sales-timegpt.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Forecasting Quarterly House Sales with TimeGPT
+sidebarTitle: House Sales with TimeGPT
+---
+
+## Introduction
+
+In this tutorial, we introduce Nixtla’s TimeGPT integration which offers the first foundational model for time series forecasting.. We’ll go through an example to predict the real estate sales.
+
+## Prerequisites
+
+### MindsDB Setup
+
+One way is to sign up for an account at [MindsDB Cloud](https://cloud.mindsdb.com/). It is a convenient option as it doesn’t require any installation procedures. You can find the details [here](/setup/cloud).
+
+Alternatively, visit our docs and follow the instructions to manually set up a local instance of MindsDB via [Docker](/setup/self-hosted/docker) or [pip](/setup/self-hosted/pip/source). You can also set up MindsDB on AWS following [this instruction set](https://aws.amazon.com/marketplace/pp/prodview-2nmhvskydmyrw).
+
+### Creating an ML Engine
+
+You can check the available engines with this command:
+
+```sql
+SHOW ML_ENGINES;
+```
+
+If you see the TimeGPT engine on the list, you are ready to follow the tutorials.
+
+## Tutorial
+
+### Connecting the Data
+
+In this tutorial, we take our [House Sales tutorial](/sql/tutorials/house-sales-forecasting) and redo it using the StatsForecast engine.
+
+We use a table from our MySQL public demo database, so let’s start by connecting MindsDB to it:
+
+```sql
+CREATE DATABASE mysql_demo_db_houses
+WITH ENGINE = 'mysql',
+PARAMETERS = {
+    "user": "user",
+    "password": "MindsDBUser123!",
+    "host": "db-demo-data.cwoyhfn6bzs0.us-east-1.rds.amazonaws.com",
+    "port": "3306",
+    "database": "public"
+};
+```
+
+Now that we’ve connected our database to MindsDB, let’s query the data to be used in the example:
+
+```sql
+SELECT *
+FROM mysql_demo_db.house_sales
+LIMIT 3;
+```
+
+Here is the output:
+
+```sql
++----------+--------------------------+-----+--------+
+|saledate  |house_price_moving_average|type |bedrooms|
++----------+--------------------------+-----+--------+
+|30/09/2007|441854                    |house|2       |
+|31/12/2007|441854                    |house|2       |
+|31/03/2008|441854                    |house|2       |
++----------+--------------------------+-----+--------+
+```
+
+The `house_sales` table stores quarterly house price moving averages per property.
+
+### Creating a Model
+
+Let's create a model table to predict the house price moving average values:
+
+```sql
+CREATE MODEL nixtla_timegpt_house_sales_predictor
+FROM mysql_demo_db
+  (SELECT * FROM house_sales)
+PREDICT house_price_moving_average
+ORDER BY saledate
+GROUP BY bedrooms, type
+WINDOW 8
+HORIZON 4
+USING ENGINE = 'timegpt';
+```
+
+The syntax is the same as in the original tutorial. But here, we add the `USING` clause that specifies the ML engine used to make predictions.
```

Comment:
We may want to add a link here, not sure if possible, but ideally at least mention the tutorial's title for reference in case the user is completely new to MindsDB.